### ISSUE TITLE
USART 3 on MKS MONSTER 8 V2

### ISSUE THREAD

**Issue by LoloMake60** [makerbase-mks/MKS-Monster8#19]
Hi,

I would like to have confirmation on the rt and tx usart3 pins.
RX = PD8 and TX = PD9.
![image](https://user-images.githubusercontent.com/100755230/183240321-a5b13b75-74f3-48b9-83bd-b3d783f6f8c9.png)

Thank you for your confirmation.

**Comment by mks-viva** [comment]
PD8 is USART3_TX, PD9 is USART3_RX

**Comment by mks-viva** [comment]
Use Klipper firmware?

**Comment by LoloMake60** [comment]
no i use marlin

**Comment by rai0012** [comment]
Uart3 works for me with marlin

**Comment by LoloMake60** [comment]
Hi,

You use usart3 for screen display?

Cordialement,


Le 31 janv. 2023 à 22:07, rai0012 ***@***.***> a écrit :

﻿

Uart3 works for me with marlin

**Comment by rai0012** [comment]
Yes, in Marlin and in Klipper.
What I can't get is the wifi to work in Marlin.

**Comment by LoloMake60** [comment]
Hi,

Maybe that's it.
![image](https://user-images.githubusercontent.com/100755230/216650157-10c61bc7-c72e-4c43-8115-63ece8b7205f.png)
For me it's works with SERIAL_PORT_1 -1 and SERIAL_PORT_2 1.

**Comment by rai0012** [comment]
But the wifi works for you

**Comment by LoloMake60** [comment]
Yes the wifi works but the driver 7 E4 it's disable because the jumper move pd11 to wifi.
![image](https://user-images.githubusercontent.com/100755230/217295041-b928374b-c84a-457c-b7f7-61ee4fa3772a.png)
I have flash my mks wifi with ESP3D.
But my screen e3v2 not works, RX and TX on PD9 and PD8. 
I don't understand!

**Comment by rai0012** [comment]
Let's clarify a bit.
I use marlin I have the dgus screen connected to uart 3 and working, I have the wifi module connected in its slot and it already works for me with flashed esp3d.
